package.json

```diff
@@ -83,6 +83,7 @@
     "@patternfly/react-core": "^3.120.5",
     "@patternfly/react-icons": "^3.14.18",
     "@rails/webpacker": "^4.2.2",
+    "@rh-uxd/integration-core": "1.0.4",
```

Comment:
You should also commit the changes in `yarn.lock`, as you are adding a new dependency